spec/factories/volunteers.rb

```diff
@@ -7,6 +7,15 @@
     waiver_signed true
     waiver_signed_at Time.zone.now
 
+    factory :volunteer_with_region do
+      after(:create) do |v|
+        region = create(:region)
+        v.regions << region
+        v.assigned = true
+        v.save
+      end
+    end
```

Comment:
I believe this added factory actually ends up doing the same thing as the `:volunteer_with_assignment` factory. Volunteers are assigned to regions through the `Assignment` model. When the assignment is created on [line 21](https://github.com/boulder-food-rescue/food-rescue-robot/pull/82/files#diff-afeda8faf6a0fa9300a3a63ab1a05925R21), a new region is also created by [this line in the Assignments factory](https://github.com/boulder-food-rescue/food-rescue-robot/blob/master/spec/factories/assignments.rb#L4).